mantidproject/mantid
# Fix issues in engg interface  - focus and calibrate

Fixes all the issues described in #16087.

**To test:**
- Use the focus and calibrate tabs with the data from previous PRs, and the new example data for ENGIN-X. All the focus and calibration actions should work well and produce all the expected output plots and files. Pick some files from the "raw data dir", but also from other locations using the "browse" buttons.
- Check carefully every of the user interface issues identified in #16087.
- Check that the status and contents of all the widgets is remembered by the interface, especially when you use the "browse" option of the MWRunFiles widgets.
- Code should look good and all tests should pass.

Fixes #16087.

All these features were either working well in the past or have been introduced in this release and should have been working well. No release notes needed.
Small additions like remembering the selected tab do not deserve a release note.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
